diagnostic_remote_logging/include/diagnostic_remote_logging/influxdb.hpp

```diff
@@ -0,0 +1,63 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2025, Willow Garage, Inc.
```

Comment:
You have to add yourself and / or your company here instead of Willow Garage. 
(Applies to all headers files with headers)